netbox_inventory/templates/netbox_inventory/inventoryitemgroup.html

```diff
@@ -79,33 +79,27 @@ <h5 class="card-header">Asset count by type & status</h5>
           <thead>
             <tr>
               <th>Inventory Item Type</th>
-              <th>Status</th>
-              <th>Count</th>
+              <th>Status - Count</th>
             </tr>
           </thead>
           <tbody>
-            {% for tsc in type_status_counts %}
+            {% for tsc in type_status_objects %}
             <tr>
-            {% ifchanged tsc.inventoryitem_type %}
               <td>
                 <a href="{% url 'plugins:netbox_inventory:inventoryitemtype' tsc.inventoryitem_type %}">
                   {{ tsc.inventoryitem_type__manufacturer__name }} {{ tsc.inventoryitem_type__model }}
                 </a>
               </td>
-            {% else %}
-              <td>&nbsp;</td>
-            {% endifchanged %}
-              <td>
-                {% badge value=tsc.label bg_color=tsc.color %}
-              </td>
-              <td>
-                <a href="{% url 'plugins:netbox_inventory:asset_list' %}?inventoryitem_type_id={{ tsc.inventoryitem_type }}&status={{ tsc.status }}">
-                  {{ tsc.count }}
-                </a>
+              <td class="d-flex">
+                {% for status in tsc.status_list %}
+                  <a href="{% url 'plugins:netbox_inventory:asset_list' %}?inventoryitem_type_id={{ tsc.inventoryitem_type }}&status={{ status.status }}" class="w-100 me-2">
+                    {% badge value=status.status|capfirst|add:' - '|add:status.count bg_color=status.color|add:' w-100' %}
```

Comment:
```suggestion
                    {% badge value=status.label|add:' - '|add:status.count bg_color=status.color|add:' w-100' %}
```

I suggest using status label here. 

At least we use localized custom status values, so displaying them consistently throughout UI is desirable.